packages/widgets/mailing-list/src/components/MailingList/MailingList.js

```diff
@@ -0,0 +1,84 @@
+import _get from 'lodash/get'
+import PropTypes from 'prop-types'
+import {useEffect} from 'react'
+import {rest, form} from 'tocco-app-extensions'
+import EntityBrowserApp from 'tocco-entity-browser/src/main'
+import {searchFormTypePropTypes} from 'tocco-entity-list/src/main'
+import {LoadMask} from 'tocco-ui'
+import {appContext as appContextPropType} from 'tocco-util'
+
+export function* modifyFormDefinition(
```

Comment:
Is it necessary to have it as generator function? can't we use a normal function in sagas?

Reply:
i think the problem is the "yield rest.fetchEntity(...)" so depending on this response the form definition is differently modified